$OpenBSD$

Disable TLSv1.3 since the LibreSSL server is not ready.
Avoid use of undeclared identifier

Index: src/tls.c
--- src/tls.c.orig
+++ src/tls.c
@@ -44,6 +44,7 @@
 #define REDIS_TLS_PROTO_TLSv1_3     (1<<3)
 
 /* Use safe defaults */
+#undef TLS1_3_VERSION
 #ifdef TLS1_3_VERSION
 #define REDIS_TLS_PROTO_DEFAULT     (REDIS_TLS_PROTO_TLSv1_2|REDIS_TLS_PROTO_TLSv1_3)
 #else
@@ -160,7 +161,7 @@ int tlsConfigure(redisTLSContextConfig *ctx_config) {
 #endif
 
 #ifdef SSL_OP_NO_CLIENT_RENEGOTIATION
-    SSL_CTX_set_options(ssl->ctx, SSL_OP_NO_CLIENT_RENEGOTIATION);
+    SSL_CTX_set_options(ctx, SSL_OP_NO_CLIENT_RENEGOTIATION);
 #endif
 
     if (ctx_config->prefer_server_ciphers)
